pkg/virt-controller/services/template.go

```diff
@@ -1121,7 +1124,7 @@ func (t *templateService) RenderExporterManifest(vmExport *exportv1.VirtualMachi
 			RestartPolicy: k8sv1.RestartPolicyNever,
 			Containers: []k8sv1.Container{
 				{
-					Name:            vmExport.Name,
+					Name:            naming.GetName("virt-export", vmExport.Name, validation.DNS1035LabelMaxLength),
```

Comment:
are container names limited?

Reply:
Yes, if the name is too long it won't pass the pod validation